cdk/l3-cdk-resources/atlas-private-endpoint/src/index.ts

```diff
@@ -0,0 +1,139 @@
+import {ApiKeyDefinition, AtlasBasic, AtlasBasicProps, ProjectProps} from "@mongodbatlas-awscdk/atlas-basic";
+import {Construct} from "constructs";
+import {
+    ApiKey,
+    CfnPrivateEndpoint,
+    CfnPrivateEndpointProps, PrivateEndpoint,
+} from "@mongodbatlas-awscdk/private-endpoint";
+
+/** @type {*} */
+const privateEndpointDefaults = {
+    region: 'us-east-1'
+};
+
+/**
+ * @description
+ * @export
+ * @class AtlasBasicPrivateEndpoint
+ * @extends {Construct}
+ */
+export class AtlasBasicPrivateEndpoint extends Construct {
+
+    readonly atlas : AtlasBasic;
+    readonly private_endpoint : CfnPrivateEndpoint;
+
+    /**
+     * Creates an instance of AtlasBasicPrivateEndpoint.
+     * @param {Construct} scope
+     * @param {string} id
+     * @param {AtlasPrivateEndpointProps} props
+     * @memberof AtlasBasicPrivateEndpoint
+     */
+    constructor(scope: Construct, id: string, props: AtlasPrivateEndpointProps) {
+        super(scope, id);
+        
+        this.atlas = new AtlasBasic(this, 'atlas-basic-'.concat(id),
+            {
+                apiKeys: props.apiKeys,
+                ...props.atlasBasicProps,
+            });
+
+        this.private_endpoint = new CfnPrivateEndpoint(this, 'private-endpoint-'.concat(id),
+            {
+                apiKeys: props.apiKeys,
+                groupId: 'this.atlas.',  //TODO
+                region: props.region || privateEndpointDefaults.region,
+                ...props.privateEndpointProps
+            });
+    }
+}
+
+/**
+ * @description
+ * @export
+ * @interface AtlasPrivateEndpointProps
+ */
+export interface AtlasPrivateEndpointProps {
+    /**
+     * @description
+     * @type {string}
+     * @memberof AtlasPrivateEndpointProps
+     */
+    readonly groupId?: string;
```

Comment:
Let's use `projectId`. we should stay consistent here, in terraform and cli we use `projectId` . 
```suggestion
    readonly projectId?: string;
```